### ISSUE TITLE
Feature: an option to not include the terminal titlebar

### ISSUE THREAD

**Issue by ilyagr** [FHPythonUtils/AnsiToImg#12]
As I was thinking more about #10, I realized that I'd prefer to not spend any space
in the README on the titlebar of the terminal (and thus not have any title).

This is similar to how your own demo in https://github.com/FHPythonUtils/AnsiToImg/blob/master/README.md#example-output currently has titles removed.

### Describe the solution you'd like

I would like the option to set `title=None` in the API and a `--no-title` option in the CLI
that removes the titlebar entirely.

### Alternative

It's likely possible to crop the image like suggested in https://github.com/FHPythonUtils/AnsiToImg/issues/7#issuecomment-1445359924, but I'd like a solution that doesn't depend on knowing the exact image size.

**Comment by FredHappyface** [comment]
Yeah I'm not sure how easy this will be to do unfortunately. Grepping the inner svg shouldn't be a problem but calculating the width and height might be a bit fiddly! I'll have a play around and see what I can do

**Comment by ilyagr** [comment]
I haven't looked into it, but it sounds like you aren't actually generating the SVG inside the tool? If `rich` does it, this might be more appropriate to implement in `rich` first.

In any case, I think this would be a nice feature, but there's absolutely no pressure or rush for you to work on it. If I feel like it, I might look into it as well, but probably not right now.

Thanks again for making a nice tool!
